examples/web-wagmi/src/profiles/UseCreateProfile.tsx

```diff
@@ -63,6 +89,10 @@ export function UseCreateProfile() {
         {error && <p>{error.message}</p>}
       </form>
 
+      {showExistingProfiles && activeProfile.data && (
```

Comment:
i think there is no harm in showing that always on the UI, can be good to showcase to our users that we properly update `useProfilesOwnedBy` and it can be relied on when building onboarding UI's

Just let's add some header: `All profiles owned by the wallet` so it's clear what the list is about.

What do you think?